geomesa-tools/examples/ingest/csv/example.csv

```diff
@@ -1,4 +1,4 @@
 ID,Name,Age,LastSeen,Friends,Lat,Lon
-23623,Harry,20,2015-05-06,"Will, Mark, Suzan",-100.236523,23
-26236,Hermione,25,2015-06-07,"Edward, Bill, Harry",40.232,-53.2356
-3233,Severus,30,2015-10-23,"Tom, Riddle, Voldemort",3,-62.23
\ No newline at end of file
+23623,Harry,20,2015-05-06,"Will, Mark, Suzan","patronus->10,expelliarmus->9",-100.236523,23
+26236,Hermione,25,2015-06-07,"Edward, Bill, Harry","accio->10",40.232,-53.2356
```

Comment:
+1 for the ref, +2 for correctness...I'm sure that @jdk2pq and @cne1x would understand the importance of correctness